datafusion/substrait/tests/roundtrip.rs

```diff
@@ -306,7 +322,7 @@ mod tests {
         explicit_options.schema = Some(&schema);
         ctx.register_csv("data", "tests/testdata/data.csv", explicit_options.clone())
             .await?;
-        ctx.register_csv("data2", "tests/testdata/data.csv", explicit_options)
+        ctx.register_csv("data2", "tests/testdata/data.csv", explicit_options.clone())
```

Comment:
Is this `clone` needed?

Reply:
You're right. The second one is not needed, since we `clone`d the first one. I also think the second call to `register_csv()` can just use the default option since we're not using the second table to test `Decimal`s so I'll fix that as well.